packages/create-plugin/templates/common/_package.json

```diff
@@ -20,8 +20,8 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@babel/core": "^7.21.4",{{#if useCypress}}
-    "@grafana/e2e": "{{ grafanaVersion }}",
-    "@grafana/e2e-selectors": "{{ grafanaVersion }}",{{/if}}
+    "@grafana/e2e": "^10.3.2",
```

Comment:
Will this create errors in the future or is renovate bot smart enough to know there is no version beyond 11.0.0? 🤔

Reply:
Looks like by default, Renovate won't go from a non-depcrecated version of a package to a deprecated version, so think we should be good. 

https://docs.renovatebot.com/configuration-options/#ignoredeprecated

Reply:
Ok cool. That means it'll be pinned to this version as the npm package (all versions) is now in a deprecated state. 👍